server/backend/database/memory/database.go

```diff
@@ -667,6 +668,47 @@ func (d *DB) CreateChangeInfos(
 	return nil
 }
 
+// DeleteOldChangeInfos delete old change infos before checpoint to save storage
+func (d *DB) DeleteOldChangeInfos(
+	ctx context.Context,
+	docInfo *database.DocInfo,
```

Comment:
We call this function with `docInfo`, but inside the function, we only use the `ID` of it.
How about only passing the ID to further reveal the intent of the function?

Reply:
I updated it!